### DISCUSSION TITLE
Change icon dropdown

### DISCUSSION THREAD

**Discussion by megabeehothanhminh** [MobileReality/react-native-select-pro#91] [General]
Hello. I can change icon Arrow?

**Comment by irekrog** [comment]
Hi @megabeehothanhminh 

Sorry, but at this moment you can't but we will add this possibility in the near future

**Comment by kkaplinski** [comment]
Hi @megabeehothanhminh

We added this feature here #93, with `customSelectControlArrowIconSource` prop.
